src/sentry/utils/registry.py

```diff
@@ -15,9 +15,10 @@ class NoRegistrationExistsError(ValueError):
 
 
 class Registry(Generic[T]):
-    def __init__(self):
+    def __init__(self, enable_reverse_lookup: bool = True):
```

Comment:
needed to do this because there are multiple detector types that could have the same config schema (e.g. empty dictionary)